tests/test_base.py

```diff
@@ -216,7 +216,6 @@ def set_up_test_model(
         # Make the "current" state.
         model_options = {
             'swappable': 'TEST_SWAP_MODEL',
-            'index_together': [['weight', 'pink']] if index_together else [],
```

Comment:
`index_together` is deprecated (dj42) and removed (dj51).
https://docs.djangoproject.com/en/5.1/releases/5.1/#features-removed-in-5-1

Redshift ignores indexes, so django-redshift-backend does not create any indexes.
I removed this definition because I left it in place when I brought it from the original Django test code.